lom/src/lib/lomcommon/config.go

```diff
@@ -223,11 +223,11 @@ type ProcPluginConfig_t struct {
 type ActionCfg_t struct {
     Name         string
```

Comment:
Can you add comments to these 2 fields too?

Reply:
added